config.go

```diff
@@ -177,6 +177,9 @@ type Config struct {
 	// The server-side message timeout for messages delivered to this client
 	MsgTimeout time.Duration `opt:"msg_timeout" min:"0"`
 
+	// Maximum size of a single message in bytes
+	MaxMsgSize int32 `opt:"max_msg_size" min:"0" default:"1048576"`
```

Comment:
I think it makes sense to have this config option, however introducing a new default size of 1M is going to be surprising for anyone upgrading `go-nsq` or using the built-ins we ship with the NSQ distribution like `nsqtofile`, etc.

I think this needs to default to `0`.

Reply:
I agree with your concern! I've set the default value of `MaxMsgSize` to 0.